### ISSUE TITLE
redesign branch is complaining about notification markers(?) being too huge

### ISSUE THREAD

**Issue by Trolldemorted** [signal-csharp/Signal-Windows#71]
```
Ausnahme ausgelöst: "System.Runtime.InteropServices.COMException" in Signal-Windows.exe
Die Benachrichtigungsmarkierung ist zu groß.

   at Windows.UI.Notifications.ToastNotification.put_Tag(String value)
   at Signal_Windows.ViewModels.MainPageViewModel.SendMessageNotification(SignalMessage message)
   at Signal_Windows.ViewModels.MainPageViewModel.HandleSignalMessage(SignalServiceEnvelope envelope, SignalServiceContent content, SignalServiceDataMessage dataMessage, Boolean isSync, Int64 timestamp)
   at Signal_Windows.ViewModels.MainPageViewModel.HandleMessage(SignalServiceEnvelope envelope)
   at Signal_Windows.ViewModels.MainPageViewModel.onMessages(SignalServiceEnvelope[] envelopes)
received message: Signal_Windows.Models.SignalMessageContent
Ausnahme ausgelöst: "System.Runtime.InteropServices.COMException" in Signal-Windows.exe
Die Benachrichtigungsmarkierung ist zu groß.

   at Windows.UI.Notifications.ToastNotification.put_Tag(String value)
   at Signal_Windows.ViewModels.MainPageViewModel.SendMessageNotification(SignalMessage message)
   at Signal_Windows.ViewModels.MainPageViewModel.HandleSignalMessage(SignalServiceEnvelope envelope, SignalServiceContent content, SignalServiceDataMessage dataMessage, Boolean isSync, Int64 timestamp)
   at Signal_Windows.ViewModels.MainPageViewModel.HandleMessage(SignalServiceEnvelope envelope)
   at Signal_Windows.ViewModels.MainPageViewModel.onMessages(SignalServiceEnvelope[] envelopes)
```

I suppose i should have a look at VS' language settings - localized error messages are not helpful if you want to google them.

**Comment by golf1052** [comment]
Apparently notification tags can only be 16 characters long https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Notifications.ToastNotification#Windows_UI_Notifications_ToastNotification_Tag

This was upped to 64 chars in the Creators Update. Will need to figure out a good way of creating short notification tags. I may just have to drop the timestamp (which could be used for navigating to a specific message) and just use the ThreadId.

**Comment by golf1052** [comment]
Fixed with a4af91e2c39cff4615da4c7bfb6d73be2267970d

**Comment by Trolldemorted** [comment]
If i understand it corectly that only the most recent notification of a certain tag will be shown? Using the threadId as the tag sounds like a good idea, so we will show only one notification (the most recent) per conversation.
